### ISSUE TITLE
`LegendList` doesn't allow passing `ref`

### ISSUE THREAD

**Issue by imranbarbhuiya** [LegendApp/legend-list#7]
In flatlist/flashlist, we can pass a ref and do scroll like
```tsx
listRef.current?.scrollToIndex({
						index,
						animated: true,
						viewOffset: 50,
});
```
but `LegendList` doesn't allow passing `ref`

**Comment by jmeistrich** [comment]
Fixed in 0.1.2 thanks to https://github.com/LegendApp/legend-list/pull/4

**Comment by imranbarbhuiya** [comment]
It does implement `scrollToIndex`, which can be added using https://react.dev/reference/react/useImperativeHandle
